storage/innobase/handler/handler0alter.cc

```diff
@@ -374,14 +374,6 @@ inline void dict_table_t::prepare_instant(const dict_table_t& old,
 			}
 		}
 
-		/* In case of discarded tablespace, InnoDB can't
-		read the root page. So assign the null bytes based
-		on nullabled fields */
-		if (!oindex.table->space) {
-			oindex.n_core_null_bytes = static_cast<uint8_t>(
-				UT_BITS_IN_BYTES(unsigned(oindex.n_nullable)));
-		}
-
```

Comment:
It would be good if the commit message explained the sequence of events that goes wrong here. (Okay, one could get that relatively easily from `./mtr --rr`.)

It would be even better to explain why this code was needed in the first place. I see that it was added in 99de8cc02878ec8c20bffeaf17443e704d6edbf7 but the commit message does not mention that.